docs/leap_sapi/sapi_rest.rst

```diff
@@ -744,37 +790,27 @@ query.\ [#]_
                            Not used for ``data`` with ``format="ref"``.
         ================== =======================================
 
-For an example of sumitting a problem to a quantum processing unit (QPU)
+For an example of submitting a problem to a quantum processing unit (QPU)
 sampler,see the :ref:`sapi_rest_full_examples` section.
 
-.. testsetup:: rest_live
```

Comment:
I see you deleted the test setup here and switched from a hybrid solver returned from Leap to a hardcoded one in line 813. Currently we're not running live tests from CI but if we start then a change to ``hybrid_binary_quadratic_model_version2`` will cause the example to fail. 
Maybe we can live fine with this.